wenet/dataset/processor.py

```diff
@@ -258,7 +258,7 @@ def compute_fbank(data,
         yield dict(key=sample['key'], label=sample['label'], feat=mat)
 
 
-def tokenize(data, symbol_table, bpe_model=None):
+def tokenize(data, symbol_table, bpe_model=None, char=False):
```

Comment:
tokenize use char as the default model unit. 
You mean your model unit is seperated with white space? it is more like phoneme, such as timit. It's better we take another name.

Reply:
I'd like to use split_with_space instead